src/labels/CustomLabels.labels

```diff
@@ -5641,6 +5657,14 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Common error when a total doesn't match Donation Amount.</shortDescription>
         <value>Total amount doesn't match the {0}</value>
     </labels>
+    <labels>
+        <fullName>geErrorCardChargedBDIFailed</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Failure description if card is charged and donation record has not been created</shortDescription>
+        <value>The card has been successfully charged but the payment has not been recorded in Salesforce. Please correct the errors and resubmit the form to record the donation in Salesforce.</value>
```

Comment:
@psadie-sfdo Oh bother. We forgot to run this past Sarah during our call today. @smarovich will you be able to make it to our sprint review tomorrow so I can get a second set of eyes on this?